code/src/Components/GetThoughts.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import moment from 'moment';
+import './getThoughts.css';
+
+// Component for getting all thoughts
+const GetThought = ({ thought, onLikesIncrease }) => {
+  return (
+    <div className="get-thoughts__wrapper">
+      <p>{thought.message}</p>
+      <div className="get-thoughts__button-wrapper">
+        <button
+          type="submit"
```

Comment:
Is the type submit necessary here?

Reply:
I did get a warning from ESlint, so doing this helped